### PR TITLE
[RFC](UN-7216] Create styles for carrousel component

### PR DESCRIPTION
**Pull request by mmelo**
Refers to: https://uniplaces.atlassian.net/browse/UN-7216

Added carroussel styles

### CODE REVIEW

File: src/styles/_components/_carroussel.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        &--next {
+            right: 0;
+        }
+    }
+
+    &__container,
+    &__container__wrapper {
+        position: relative;
+        height: 100%;
+        transform: translate3d(0px,0px,0);
```
Comment (AndreJoaquim) [inline src/styles/_components/_carroussel.scss]: This needs the 'px' ?

File: src/styles/_components/_carroussel.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        transition: transform $animation-duration $swiftOut;
```
Comment (AndreJoaquim) [inline src/styles/_components/_carroussel.scss]: I am not fully aware of the conventions followed here ^^' but it seems inconsistent to have some vars dasherized and some camelcased

File: src/styles/_components/_carroussel.scss [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    &__container__wrapper {
+        position: relative;
+        height: 100%;
+        transform: translate3d(0px,0px,0);
+        transition: transform $animation-duration $swiftOut; 
+        will-change: transform;
+
+        &__item {
+            position: relative;
+            display: inline-flex;
```
Comment (AndreJoaquim) [inline src/styles/_components/_carroussel.scss]: Just a question: will this have problems in Safari? Don't we need -webkit-* ?

File: src/styles/_components/_carroussel.scss (hunk 1 above)
Comment (mmelo) [inline src/styles/_components/_carroussel.scss]: yes, as we are creating a "motion-tween" it's always a good practice to set units so the browser does not need to "figure" them out.

File: src/styles/_components/_carroussel.scss (hunk 2 above)
Comment (mmelo) [inline src/styles/_components/_carroussel.scss]: agreed, this is legacy stuff from previous implementations.

**Comment by mmelo** [comment]
@AndreJoaquim @raffo0707 can I merge this?

File: src/styles/_components/_carroussel.scss (hunk 1 above)
Comment (AndreJoaquim) [inline src/styles/_components/_carroussel.scss]: 👍

**Pull request merged**